app/src/main/java/com/lorenzovainigli/foodexpirationdates/view/composable/screen/MainScreen.kt

```diff
@@ -82,14 +100,16 @@ fun MainScreen(
     }
 }
 
+@SuppressLint("UnrememberedMutableState")
 @RequiresApi(Build.VERSION_CODES.O)
 @Preview
 @Composable
 fun MainScreenPreview() {
     FoodExpirationDatesTheme {
         Surface {
             MainScreen(
-                navController = rememberNavController()
+                navController = rememberNavController(),
+                searchQuery = mutableStateOf("")
```

Comment:
You can remove it after introducing the default value.
```suggestion
                navController = rememberNavController()
```